docusaurus.config.js

```diff
@@ -0,0 +1,111 @@
+// @ts-check
+
+const GITHUB_ORG = "comcode-org";
+const GITHUB_PROJECT = "hackmud_wiki";
+
+// See: https://docusaurus.io/docs/api/docusaurus-config
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  /**
+   * GENERAL CONFIG
+   */
+  // Metadata
+  title: "hackmud Wiki",
+  tagline: "Knowledge repository for the video game hackmud",
+  //favicon: 'img/favicon.ico',
+
+  // Deployment details
+  url: `https://${GITHUB_ORG}.github.io`,
+  baseUrl: `/${GITHUB_PROJECT}/`,
+
+  // GitHub Pages config for CLI deployment
+  organizationName: GITHUB_ORG,
+  projectName: GITHUB_PROJECT,
+
+  // Fail loudly on broken links
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+
+  // Internationalization options, useful to set even if you only support one
+  // language because it generates the `lang` attribute on the `html` tag
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"],
+  },
+
+  /**
+   * INSTALLED PLUGINS
+   */
+  plugins: [
+    [
+      "@docusaurus/plugin-content-docs",
+      // See: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#configuration
+      {
+        // Path of the docs plugin relative to the root. Since it's the only
+        // plugin we're using, put it at the top
+        routeBasePath: "/",
+        // Base url for "Edit This Page" button on content pages
+        // TODO: decide whether to accept *content* edits directly to `main`,
```

Comment:
we need to track these todos outside of the source code. 
@kbinreallife FYI opportunity to get us set up with task tracking here.

Reply:
Should I remove TODOs and keep track in a document in the P003 folder for now?

Reply:
We already have a todo document up if you'd like to add things to it.

Reply:
Moved TODOs to the appropriate document, and removed this line in https://github.com/comcode-org/hackmud_wiki/pull/2/commits/0085879e3fc4a073af024a95febfd5dfe1569ab3.

Reply:
Once we have task tracking, you would leave the TODO and reference the task id number or link in your comment, making it easier to find either of them. For now deleting the comment is good.